### ISSUE TITLE
Fonts served from wrong path

### ISSUE THREAD

**Issue by sevmonster** [gary-kim/riotchat#442]
I noticed that fonts are being served from the root of the site instead of relatively, leading to 404's. Is this an Element bug or for here?

Release Element index.html:
![image](https://user-images.githubusercontent.com/16788697/145173144-4d0e554e-7b06-4228-818f-fb2905e7ac59.png)

Defined here:
https://github.com/vector-im/element-web/blob/996d79847ab6f2dcc1e060cc249c9739c1cf21dc/src/vector/index.html#L56

Fix in nginx (using the community-maintained config):
``` nginx
    location ~ \.woff2?$ {
        # BUG: fix /fonts in riotchat
        rewrite ^/fonts/ /apps/riotchat/riot$request_uri permanent;
```

**Comment by gary-kim** [comment]
Good catch. Whether it is an upstream issue would depend on whether serving in a subdomain is an officially supported setup which to be honest, I don't know.
